pkg/compiler/compiler.go

```diff
@@ -263,7 +263,7 @@ func CompileAndSave(src string, o *Options) ([]byte, error) {
 			for j, p := range e.Parameters {
 				params[j] = DebugParam{
 					Name: p.Name,
-					Type: p.Type.String(),
+					Type: p.Type.ConvertToStackitemType().String(),
```

Comment:
And BTW, this is a bag and probably this fix needs to be added as a separate PR. @roman-khimov, do we need it?

Reply:
Hmm. https://github.com/neo-project/proposals/blob/master/nep-19.mediawiki#typename

Reply:
We have several types mismatch in our code and a lot of them in our tests, I'll create a separate PR to fix that. I alsoo need to re-check how debugger works with it.